app/utils/transactionCosts.ts

```diff
@@ -125,24 +131,38 @@ function getEnergyCostOfType(transactionKind: TransactionKindId) {
     }
 }
 
-export function getScheduledTransferPayloadSize(scheduleLength: number) {
+export function getScheduledTransferPayloadSize(
+    scheduleLength: number,
+    memoLength?: number
```

Comment:
This is never called without `memoLength` being set.

Reply:
Some tests did, but I changed that.